RadarSDK/RadarAPIClient.m

```diff
@@ -326,6 +326,34 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
         }
     }
     params[@"appId"] = [[NSBundle mainBundle] bundleIdentifier];
+    RadarSdkConfiguration *sdkConfiguration = [RadarSettings sdkConfiguration];
+    if (sdkConfiguration.useLocationMetadata) { 
+        NSMutableDictionary *locationMetadata = [NSMutableDictionary new];
+        locationMetadata[@"motionActivityData"] = [RadarState lastMotionActivityData];
+        locationMetadata[@"accelerometerData"] = [RadarState lastAccelerometerData];
+        locationMetadata[@"gyroData"] = [RadarState lastGyroData];
+        locationMetadata[@"magnetometerData"] = [RadarState lastMagnetometerData];
+        locationMetadata[@"heading"] = [RadarState lastHeadingData];
```

Comment:
How do we want to send these up if they're not present (e.g. if motion activity permission wasn't granted or, in a build before `RadarSDKMotion` was added to the client app)

Reply:
this is actually a good point, I do feel that given the labels we are applying to these values, we should ensure that these values are actually "fresh". I think we should actually return null values if the data is more than 1 min old or something like that.

Reply:
Right now if the values are not available, we are assigning null to the field and the serialization gets rid of the field, which I think is the right thing to do.

Thinking about json and webhookland, having a field with value "null" just to keep the shape might not be the best pattern. It may be more user friendly to just not have the field since `{}.nonExistentField` evaluates to the null value in most languages anyways. Having a non null value of the string "null" may throw people off more, especially since that is not a pattern we have elsewhere.